apollo-sqlite-cache/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=apollo-sqlite-cache
```

Comment:
Nitpicking about the naming: what about `apollo-normalized-cache-sqldelight` ?  This would:

* put the module next to `apollo-normalized-cache` and `apollo-normalized-cache-api` when sorted alphabetically
* make a clear distinction between 'normalized' caches and 'http' ones.
* allow for sqlite implementation without sqldelight

Reply:
What about `apollo-normalized-cache-sqlite`?

I genuinely think that SqlDelight should be implementation detail. We provide persisted normalized cache backed by `SQLite`.

- We wouldn't provide 2nd implementation side by side
- If we want to replace it in the future, it's easier (not exposed)
- If a user wants to have a different implementation, they can always implement `NormalizedCache` from the API

Reply:
Sounds good!